packages/nextjs/src/config/wrappers/withSentryServerSideGetInitialProps.ts

```diff
@@ -13,10 +13,7 @@ type GetInitialProps = Required<NextPage>['getInitialProps'];
  * @param parameterizedRoute The page's parameterized route
  * @returns A wrapped version of the function
  */
-export function withSentryServerSideGetInitialProps(
-  origGetInitialProps: GetInitialProps,
-  parameterizedRoute: string,
-): GetInitialProps {
+export function withSentryServerSideGetInitialProps(origGetInitialProps: GetInitialProps): GetInitialProps {
```

Comment:
Oh, maybe this will solve the problem I was having [here](https://github.com/getsentry/sentry-javascript/pull/5602/files#diff-dd9272eec5c418e22f1d78d2044ef7e0e31c1fa6ecfd759373efc478ebd01575R29-R34), which forced me to do the typecast. I spent entirely too much time trying to bend TS to my will on that one and failed entirely.